__tests__/actions/user.js

```diff
@@ -0,0 +1,95 @@
+import moxios from 'moxios';
```

Comment:
The tests are valid and are what I have used. There isn't anything wrong with them.